src/py-opentimelineio/opentimelineio/plugins/manifest.py

```diff
@@ -7,13 +7,9 @@
 import inspect
 import logging
 import os
-from pathlib import Path
 
-try:
-    from importlib import metadata
-except ImportError:
-    # For python 3.7
-    import importlib_metadata as metadata
+# for python <= 3.9
```

Comment:
does this comment tell us we will delete it when we drop 3.9?

Reply:
Yeah, exactly